lib/classes/Service.js

```diff
@@ -33,6 +33,8 @@ class Service {
   load(opts) {
     const that = this;
     const options = opts || {};
+    options.stage = options.stage || options.s;
```

Comment:
and plugin manager depends on `serverless.yml` to be loaded so that custom plugins (and their options) are loaded and then validated against the cli options.